app/services/Config.scala

```diff
@@ -65,7 +68,6 @@ sealed trait Config {
   }
 
   lazy val permissionsStage: String = readTag("Stage") match {
-    case Some("DEV") =>  "CODE"
```

Comment:
But this line is being removed? Won't this mean if someone wants to run in DEV they will have to run a permissions locally?

Sorry, I'm a bit confused :cry: